torch/quantization/fx/graph_module.py

```diff
@@ -47,8 +44,34 @@ def __deepcopy__(self, memo):
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
         return ObservedStandaloneGraphModule(fake_mod, self.graph)
 
-def mark_observed_standalone_module(module: GraphModule) -> GraphModule:
-    return ObservedStandaloneGraphModule(module, module.graph)
-
 def is_observed_standalone_module(module: Any) -> bool:
     return isinstance(module, ObservedStandaloneGraphModule)
+
+
+class QuantizedGraphModule(GraphModule):
+    def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph):
+        super().__init__(root, graph)
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        for attr_name in dir(self):
+            if "_packed_weight" in attr_name and \
```

Comment:
I am not familiar with the quantization process but is "weight" always used here? Or can it be anything else?

Reply:
do you mean the "_packed_weight" name?

Reply:
Yes.
In particular, the `_packed` part sounds like it can be common, but I wasn't sure about the weight?
Also you might want to match the beginning of the string here as well to avoid matching things like "my_custom_packed_weight".

Reply:
sure, I have the same concerns, I can change that to a startswith check